src/services/db/models.py

```diff
@@ -32,18 +32,23 @@ class CatalogueItem(db.Model):
     checksum_value = db.Column(db.String)
 
     transfer_id = db.Column(db.String)
-    transfer_status = db.Column(db.String, default=TransferStatus.NOT_STARTED.value)
+    transfer_status = db.Column(
+        db.String, default=TransferStatus.NOT_STARTED.value, index=True
+    )
     transfer_checksum_value = db.Column(db.String, nullable=True)
     transfer_checksum_verification = db.Column(db.String(20), nullable=True)
     transfer_started_on = db.Column(db.DateTime, nullable=True)
     transfer_completed_on = db.Column(db.DateTime, nullable=True)
     transfer_source = db.Column(db.String, nullable=True)
     transfer_destination = db.Column(db.String, nullable=True)
 
-    sealed_state = db.Column(db.String)
+    sealed_state = db.Column(db.String, index=True)
```

Comment:
What's a sealed_state?

Reply:
The sealed state status indicates the file storage location. There are three sealed states. A sealed state indicates the file is in the cold storage/archived state. An unsealed state means that the file is in the hot storage state and readily available to be transferred. Unsealing state means that the file is being transitioned from cold storage to hot storage.